typeshed

```diff
@@ -1 +1 @@
-Subproject commit 0e5003b61e7c0aba8f158f72fe1827fd6dcd4673
+Subproject commit f90a6d1c0eeda7b2b267d9bbfcc6681f4d9f8168
```

Comment:
It always happens. I don't really know why, whether it's OK, and how to avoid it (I often use `commit -a -m'...'`).